source/blog/2016-09-05-last-week-in-manageiq-api-refactor-observables.html.md

```diff
@@ -0,0 +1,81 @@
+
+### [Fix enabling of toolbar components if item is selected in automate explorer](https://github.com/ManageIQ/manageiq/pull/10827)
+[Karel Hála](https://github.com/karelhala) had recently [converted our toolbars to angular](https://github.com/ManageIQ/manageiq/pull/9753) and now he's fixing the odd places where that didn't quite work out, this time in Automate. Thanks for touching that :).
+
+### [Add missing policy buttons to Network Manager Relationship pages](https://github.com/ManageIQ/manageiq/pull/10584)
+One of our newest colleagues, [Hilda Šťastná](https://github.com/hstastna) has added a Policy toolbar button to all the places that should have it but didn't. More tags for everyone :cookie:!
+
+
+## New
+
+### [Archived: Scope for not archived nor orphaned](https://github.com/ManageIQ/manageiq/pull/10920)
+[Keenan Brock](https://github.com/kbrock), our DB guru, has found an *old* TODO and has actually TODONE. (TODID?)
+Now, we have a scope for non-archived, non-orphaned VMs.
+
+### [Authentication token for initiation web sockets](https://github.com/ManageIQ/manageiq/pull/10874)
+[Šimon Lukašík](https://github.com/isimluk) has been working on making the websocket have its own authentication token, with some [prior TokenManager changes](https://github.com/ManageIQ/manageiq/pull/10808). This being done, we should be well on the way towards asynchronous notifications via websockets, wheee.
+
+### [API - add backend version info](https://github.com/ManageIQ/manageiq/pull/10848)
+People really seem to like our About dialog, so we're bringing it to the Self-Service UI too. Just a small change from yours truly, adding some necessary info to the API entrypoint.
+
+
+## Deleted
+
+### [Remove process_tasks methods from provider classes](https://github.com/ManageIQ/manageiq/pull/10985)
+[Nick Carboni](https://github.com/carbonin) has found some more dead code, this time `process_tasks` in a couple of provider classes .. aaand it's gone, never to be seen again :).
+
+### [Remove old "admintable" class table styling](https://github.com/ManageIQ/manageiq/pull/10921)
+[Eric Winchell](https://github.com/epwinchell) has been working on multiple old styling cleanups - [admintable class](https://github.com/ManageIQ/manageiq/pull/10921), [tagstable class](https://github.com/ManageIQ/manageiq/pull/10881), [form buttons](https://github.com/ManageIQ/manageiq/pull/10878) and [checkboxes](https://github.com/ManageIQ/manageiq/pull/10876). Good riddance.
+
+### [Removed unused report tree partials](https://github.com/ManageIQ/manageiq/pull/10790)
+[David Halász](https://github.com/skateman) has also found a couple of unused tree partials during his bootstrap tree work - and he promptly dispatched them to `/dev/null`.
+
+
+## Wrapping up
+
+Of course I missed most of the boring PRs doing some actual work, but feel free to check all of them out [here][PRs merged last week]. And stay tuned for even more exciting news next week :).
```

Comment:
I feel a bit uneasy about saying "boring" and "actual work" here, unless I'm misreading this?